app/models/concerns/pagination.rb

```diff
@@ -89,6 +97,16 @@ def position_within(collection)
     (collection.index(self) + 1).ordinalize
   end
 
+  # @return [Boolean]
+  def feedback_form?
+    parent.feedback_questions.first.eql?(self)
+  end
+
+  # @return [Boolean]
```

Comment:
@martikat this is wrong and I have removed it but it'd be revealing if you could tell me why

Reply:
This is a something I was planning on updating, but missed when pushing my changes up